### PR TITLE
chore: google-site-verification meta tag

### PR DESCRIPTION
**Pull request by awlevin**
Adds Google site verification tag so we can see how open.fiveable.me performs in Google search results.

### CODE REVIEW

File: pages/_document.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import Document, { Head, Html, Main, NextScript } from "next/document";
+import React from "react";
```
Comment (sarthaktexas) [inline pages/_document.js]: why are we importing react

**Review by sarthaktexas — CHANGES_REQUESTED** [review]

File: pages/_document.js (hunk 1 above)
Comment (awlevin) [reply pages/_document.js]: ok bye

Comment (sarthaktexas) [reply pages/_document.js]: i dont even know how the linter even allowed you to commit this

Comment (awlevin) [reply pages/_document.js]: hahhaha. fixed 👍

**Review by sarthaktexas — APPROVED** [review]

**Review by exu3 — APPROVED** [review]

**Pull request merged**